### PR TITLE
Add build workflow

### PR DESCRIPTION
**Pull request by wahlp**
Automatically build executable for Windows

### CODE REVIEW

**Comment by wahlp** [comment]
didn't expect it to be merged immediately, unless you wanted to see it for yourself?

the steps in `build.yml` were made as a simple proof of concept for using github actions. it would make a new release automatically (tagged using current date and time) with the zipped executable attached.

i didn't figure out how to follow the current tag naming strategy currently used.

**Comment by AnthonyMichaelTDM** [comment]
I have some ideas, I'll see if I can make them work

**Comment by AnthonyMichaelTDM** [comment]
let's see if that works, should run the workflow when a new tag is created

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Build Windows Executable
+
+on:
+  push:
+    tags:
+      - "v*"
+    branches:
+      - master
+      - executable
```
Comment (AnthonyMichaelTDM) [inline .github/workflows/build.yml:9]: @wahlp can you try creating a tag (name doesn't matter, as long as it starts with "v") on your fork to see if the workflow triggers? if it does, we can remove this line and merge

Comment (wahlp) [reply .github/workflows/build.yml:9]: created a tag, pushed it, triggered: https://github.com/wahlp/DRG-Save-Editor/releases/tag/v0

Comment (AnthonyMichaelTDM) [reply .github/workflows/build.yml:9]: perfect, thanks

**Pull request merged**